OrchardHUN.TrainingDemo.Tests/StartHere.txt

```diff
@@ -13,7 +13,7 @@ Other gotchas:
 
 If you don't really know what unit testing is about please study the topic first.
 
-There is no scaffolding command to create unit test projects (yet: http://orchard.codeplex.com/workitem/18696) so you have to create them by hand.
+There is no scaffolding command to create unit test projects (yet: https://www.google.com/search?q=Work+Item+Codeplex) so you have to create them by hand.

```

Comment:
Instead of this URL, you can link this URL that contains the https://github.com/OrchardCMS/Orchard/blob/dev/src/Orchard.Web/Modules/Orchard.CodeGeneration/Commands/CodeGenerationCommands.cs#L134 the given scaffolding command. And of course, you need to rephrase this to something like:

You can use the codegen moduletests scaffolding command to create unit test projects (https://github.com/OrchardCMS/Orchard/blob/dev/src/Orchard.Web/Modules/Orchard.CodeGeneration/Commands/CodeGenerationCommands.cs#L134).

And you can remove the rest of this paragraph.